### PR TITLE
Maintenance related to KO websites 13

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/78474
 - https://github.com/AdguardTeam/AdguardFilters/issues/27999
 - https://github.com/AdguardTeam/AdguardFilters/issues/25255
 - https://github.com/AdguardTeam/AdguardFilters/issues/23846
 - And, tistory-related rules in Base and Annoyances filter.

### Add your comment and screenshots

 - https://github.com/AdguardTeam/AdguardFilters/issues/78474 - The website is dead.
 - https://github.com/AdguardTeam/AdguardFilters/issues/27999 - The anti-adblock script does not appear on my side.
 - https://github.com/AdguardTeam/AdguardFilters/issues/25255, https://github.com/AdguardTeam/AdguardFilters/issues/23846 - Resolved by List-KR filter.
 

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/foreign.txt [hunk 1]
```diff
@@ -4788,8 +4788,6 @@ tv.kakao.com,namu.wiki,daum.net#@##adContainer
 @@||videoads.kakao.com/adserver/api/*/vmap$domain=tv.kakao.com|namu.wiki|daum.net
 @@||videoads.kakao.com/adserver/api/*/vast$domain=tv.kakao.com|namu.wiki|daum.net
 !#endif
-! https://github.com/AdguardTeam/AdguardFilters/issues/78474
-dajunbi.com##.basic-banner
```
Comment (zloyden) [inline BaseFilter/sections/foreign.txt]: Don't remove this rule, it's still relevant.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/foreign.txt (hunk 1 above)
Comment (zloyden) [reply BaseFilter/sections/foreign.txt]: Btw, the original rule
```
joytv.fun,dajunbi.com##.basic-banner
```

Comment (zloyden) [reply BaseFilter/sections/foreign.txt]: Ok, what about this site `joytv.fun`? Is it opened on your end?

Comment (piquark6046) [reply BaseFilter/sections/foreign.txt]: `joytv.fun` is a dead domain.

**Review by zloyden — APPROVED** [review]

File: BaseFilter/sections/foreign.txt (hunk 1 above)
Comment (piquark6046) [reply BaseFilter/sections/foreign.txt]: https://github.com/AdguardTeam/AdguardFilters/commit/5a7c269a72daf1279afa944c9a250128125159d1

Comment (zloyden) [reply BaseFilter/sections/foreign.txt]: I can't, that's why I asked.

**Pull request merged**